lib/appbundler/cli.rb

```diff
@@ -9,10 +9,12 @@ class CLI
     banner(<<-BANNER)
 * appbundler #{VERSION} *
 
-Usage: appbundler APPLICATION_DIR BINSTUB_DIR
+Usage: appbundler BUNDLE_DIR BINSTUB_DIR [GEM_NAME] [GEM_NAME] ...
```

Comment:
backcompat might also be tricky here in order to keep appbundle-updater working with older omnibus versions (which you still need to support upgrading with appbundle-updater)